solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/impl/TestLambdaMARTModel.java

```diff
@@ -127,60 +127,121 @@ public void lambdaMartTestExplain() throws Exception {
     System.out.println(restTestHarness.query("/query" + query.toQueryString()));
   }
 
-  @Test(expected = ModelException.class)
+  @Test
   public void lambdaMartTestNoParams() throws Exception {
-    createModelFromFiles("lambdamart_model_no_params.json",
-        "lambdamart_features.json");
+    final ModelException expectedException = 
+        new ModelException("Model type does not exist org.apache.solr.ltr.ranking.LambdaMARTModel");
+    try {
+        createModelFromFiles("lambdamart_model_no_params.json",
+              "lambdamart_features.json");
+    } catch (ModelException actualException) {
+      assertEquals(expectedException.toString(), actualException.toString());
+    }
 
   }
 
-  @Test(expected = ModelException.class)
+  @Test
   public void lambdaMartTestNoTrees() throws Exception {
-    createModelFromFiles("lambdamart_model_no_trees.json",
-        "lambdamart_features.json");
+    final ModelException expectedException = 
+        new ModelException("Model type does not exist org.apache.solr.ltr.ranking.LambdaMARTModel");
+    try {
+        createModelFromFiles("lambdamart_model_no_trees.json",
+            "lambdamart_features.json");
+        fail("unexpectedly got here instead of catching "+expectedException);
+    } catch (ModelException actualException) {
+      assertEquals(expectedException.toString(), actualException.toString());
+    }
   }
 
-  @Test(expected = ModelException.class)
+  @Test
   public void lambdaMartTestNoWeight() throws Exception {
-    createModelFromFiles("lambdamart_model_no_weight.json",
-        "lambdamart_features.json");
+    final ModelException expectedException = 
+        new ModelException("Model type does not exist org.apache.solr.ltr.ranking.LambdaMARTModel");
+    try {
+        createModelFromFiles("lambdamart_model_no_weight.json",
+            "lambdamart_features.json");
+        fail("unexpectedly got here instead of catching "+expectedException);
```

Comment:
Should likely say something "lambdaMartTestNoWeight failed to throw exception: " + expectedException